circleci/images/Makefile

```diff
@@ -87,6 +88,34 @@ endef
 # call make-image-targets($PG_VERSION, $PG_MAJOR) for every version in PG_VERSIONS
 $(foreach element,$(PG_VERSIONS),$(eval $(call make-image-targets,$(element),$(shell echo $(element) | awk -F'[^0-9]*' '/[0-9]/ { print $$1 }'),$(shell echo $(element) | sed 's/~//'))))
 
+define make-image-targets-source
+# $1 = PG_VERSION
+# $2 = PG_MAJOR
+# $3 = PG_VERSION_CLEAN
+# $4 = BUILD_MODE
+# $5 = PG_BRANCH
+
+build-extbuilder-$1:
```

Comment:
We have a similar line in `make-image-targets` that will be overridden by this line here. Do you think that it is safe to have them both?

My guess is that we should have only one of the two; compile from tag, or source. Maybe we can add a comment before the definitions of `PG_VERSIONS` and `PG_SOURCE_VERSIONS` at the top to make sure they never contain the same major in both variables.

Reply:
They both contain `$1` which gets substituted with the actual versions interpolated. The actual target for the 15 source build becomes `build-extbuilder-15~~devel` for that matter.